cgo/pow/pow.go

```diff
@@ -36,6 +40,15 @@ func init() {
 	C.dcurl_init_wrapper()
 	proofOfWorkFuncs["Dcurl"] = DcurlProofOfWork
 	proofOfWorkFuncs["SyncDcurl"] = SyncDcurlProofOfWork
+
+	logger := log.New(os.Stdout, "WARN", log.Ldate | log.Ltime)
+	gracefulShutdown := make(chan os.Signal)
+	signal.Notify(gracefulShutdown, syscall.SIGTERM, syscall.SIGINT)
+	go func() {
+		<- gracefulShutdown
+		logger.Println("test")
```

Comment:
Remove testing message.

Reply:
It has been modified to warning message.
Thanks for your  review!